sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/DoNotDecreaseMemberVisibility.cs

```diff
@@ -100,9 +100,15 @@ public Diagnostic FindIssue(MemberDeclarationSyntax memberDeclaration)
                 var methodSymbol = memberSymbol as IMethodSymbol;
                 if (methodSymbol != null)
                 {
+                    var methodDeclaration = memberDeclaration as MethodDeclarationSyntax;
+                    if (methodDeclaration?.Modifiers.Any(m => m.ToString() == "new") == true)
```

Comment:
Argh! Use `IsKind(SyntaxKind.NewKeyword)` instead. Also the `== true` when using `.Any()` first is pointless.